pkg/virt-handler/vm_test.go

```diff
@@ -1684,15 +1685,14 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Expect(len(arg.(*v1.VirtualMachineInstance).Status.Interfaces)).To(Equal(1))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].Name).To(Equal(podCacheInterface.Iface.Name))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IP).To(Equal(podCacheInterface.PodIP))
-				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs[0]).To(Equal(podCacheInterface.PodIP))
-
+				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs).To(Equal(podCacheInterface.PodIPs))
```

Comment:
This checks equality or identity? Don't you need to use `DeepEqual`? Or is `Equal` does it for us in the implementation?

Reply:
according Ram Equal use DeepEqual behind the scene and check for equality
@RamLavi right ?

Reply:
Yes, I see it is commented on the API of the function. Thanks.